codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -176,15 +175,72 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
     }
 
+    @Override
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
The current implementation looks appropriate, if that's what you're asking. Even though it seems like could have just used `ctx.fireExceptionCaught()` initially, I made sure to always go through the handler since which methods were overridden changes over time and fixing up all callers is really error prone.

I expect with #4914 we could start using ctx directly because the handler will become so much simpler and we won't be needing to override very many outbound methods (if any).